### ISSUE TITLE
Keybinding to toggle theme for Linux users

### ISSUE THREAD

**Issue by wirehiker** [freetonik/vscode-dnts#11]
Ctrl+Alt+t is in most of the Linux distribution a keybinding already opens the Terminal program. It would be great to have an alternative keybinding like Ctrl+Super+t for Linux users.

**Comment by pnchinmay** [comment]
> Ctrl+Alt+t is in most of the Linux distribution a keybinding already opens the Terminal program. It would be great to have an alternative keybinding like Ctrl+Super+t for Linux users.

Alright, I found the alternate
Just go in `File -> Preferences -> Keyboard Shortcuts`
Search for `toggleDayNightTheme`
change the keybinding and you're done

**Comment by wirehiker** [comment]
> 

Obviously, already done a long time ago, but thank you. I just wanted to report my issue, so maybe other users can benefit from it out of the box after the default keybinding gets updated, whenever possible...

**Comment by pnchinmay** [comment]
When I came looking for this problem, I couldn't find the solution. So, when I found one, thought to share with other users who may come looking for alternate keybindings.

**Comment by pnchinmay** [comment]
When I came looking for this problem, I couldn't find the solution. So, when I found one, thought to share with other users who may come looking for the same.
